src/ServiceResolverOptions.php

```diff
@@ -11,11 +13,16 @@ private function __construct()
     {
     }
 
-    public static function create(): self
+    public static function empty(): self
     {
         return new self();
     }
 
+    public function build(): ServiceResolver
```

Comment:
I found it a little surprising that the `Options->build()` method creates an instance of the `Resolver` itself. I don't think that's a problem, it's just a pattern that I don't remember having seen before.

Reply:
If I am not mistaking, `ServiceResolverOptions` is not following the _Builder Design Pattern_.

Reply:
Please, follow the [other discussion above](https://github.com/bauhausphp/service-resolver/pull/10#discussion_r602725822).